geoportailv3/static/js/olcs/lux3dmanager.js

```diff
@@ -30,24 +30,35 @@ app.olcs.Lux3DManager = class extends ngeo.olcs.Manager {
     this.luxCameraExtentInRadians = cameraExtentInRadians;
   }
 
-
   /**
    * @override
    */
-  configureForUsability(scene) {
-    super.configureForUsability(scene);
-    const camera = scene.camera;
-    camera.constrainedAxisAngle = 7 * Math.PI / 16; // almost PI/2
+  instantiateOLCesium() {
+    goog.asserts.assert(this.map);
+    const terrainExaggeration = parseFloat(this.ngeoLocation_.getParam('terrain_exaggeration') || '2.0');
```

Comment:
Luxembourg is actually using a StateManager

Reply:
This is not a state of the application (not changeable by the user), but an external configuration. We never want to get the value from the local storage.